### ISSUE TITLE
npm issues related to nvm

### ISSUE THREAD

**Issue by chapgaga** [coreybutler/nvm-windows#55]
all things works well with 0.10.25, but when i use 0.10.34, npm search not works, but npm info still work, what's wrong with nvm?

D:>nvm list
0.12.1
0.10.36

0.10.34 (Currently using 32-bit executable) 0.10.25
D:>npm --version
1.4.28

**Comment by coreybutler** [comment]
Nvm has nothing to do with how npm works. This is most likely an issue with an older version of npm.

**Comment by chapgaga** [comment]
failed to search even with latest npm, why?
![](http://img.itc.cn/photo/jfetdhUeWBP)
http://img.itc.cn/photo/jfetdhUeWBP.jpg

**Comment by coreybutler** [comment]
I don't know why you're seeing this response, but I do know it is not related to NVM. When I run `npm search express` using node 0.12 it built the local index and displayed the results as expected. NVM merely installs npm by downloading it to the appropriate directory. NVM doesn't touch npm's functionality... it's can't, nvm is a Go app, while npm is a node.js app. The run in separate runtimes. 

`No match found for "____"` is the result npm generates when the remote registry responds with no results. This means the local code is executing properly, but the remote npm registry isn't responding with results. Check your firewalls, make sure you're .npmrc isn't trying to use a private registry, double check any proxy configurations, etc. 

When I ran the exact same command, I saw hundreds of responses for Express. Bottom line, this isn't an NVM issue. The guys over at the [npm](https://github.com/npm/npm) may be able to help you though.
